src/output-lua.c

```diff
@@ -823,8 +825,8 @@ static OutputInitResult OutputLuaLogInit(ConfNode *conf)
         } else if (opts.alproto == ALPROTO_SSH) {
             om->TxLogFunc = LuaTxLogger;
             om->alproto = ALPROTO_SSH;
-            om->tc_log_progress = SshStateBannerDone;
-            om->ts_log_progress = SshStateBannerDone;
+            om->tc_log_progress = SshStateFinished;
```

Comment:
Maybe `SSHTxLogCondition` should be used here (this needs testing)

Reply:
Yes, my bad.